pyannote/audio/core/task.py

```diff
@@ -201,11 +256,25 @@ class Task(pl.LightningDataModule):
     ----------
     specifications : Specifications or tuple of Specifications
         Task specifications (available after `Task.setup` has been called.)
+
+    Notes
+    -----
+
+    task(protocol, cache="cache.npz")
+    model = MyModel(task=task)
+
+    model.setup()
+    task.prepare_data()
```

Comment:
This note section should either be extended with better instructions or removed.